.github/workflows/ad_hoc_dev_deploy.yml

```
@@ -0,0 +1,54 @@
+name: Deploy branch to dev
+run-name: Deploy branch '${{github.ref_name}}' to dev
+
+permissions:
+  contents: read  # This is required for actions/checkout
+  id-token: write  # This is required for authenticating with aws
+
+on:
+  workflow_dispatch:
+    inputs:
+        run_e2e_tests:
+            required: false
+            default: true
+            type: boolean
+            description: Run e2e tests
+  schedule:
+  - cron:  '0 9 * * 1'
```

```suggestion
  - cron:  '0 2 * * 1'
```

just shifting it into the middle of the night because 9am feels awkward for me/people who start before then